src/js/redux/reducers.ts

```diff
@@ -250,6 +250,19 @@ export function populateQuickstartsReducer(
   };
 }
 
+export function addQuickstartstoApp(state: ChromeState, { app, quickstart }: { app: string; quickstart: QuickStart }) {
```

Comment:
Can you write some tests for the function?

Reply:
done